hw/chisel/src/main/scala/snax/xdma/xdmaExtension/DMAExtension.scala

```diff
@@ -98,3 +97,73 @@ abstract class DMAExtension(implicit extensionParam: DMAExtensionParam)
   // When bypass is 1, io.in(1) is connected to bypass signal
   outputMux.io.in(1) <> bypass_data
 }
+
+/** The parent Class for the integration of DMA Extension written in
+  * SystemVerilog Before the integration of the custom extension, please make
+  * sure the following: 1) The SystemVerilog module's top module's IO is strict,
+  * shown as the following example:
+  *
+  * module VerilogMemset #( userCsrNum = 1, dataWidth = 512 ) ( input logic clk,
+  * input logic rst_n, output logic ext_data_i_ready, input logic
+  * ext_data_i_valid, input logic [dataWidth-1:0] ext_data_i_bits, input logic
+  * ext_data_o_ready, output logic ext_data_o_valid, output logic
+  * [dataWidth-1:0] ext_data_o_bits, input logic [31:0]ext_csr_i_0, input logic
+  * ext_start_i, output logic ext_busy_o ); Several reminders for developers:
```

Comment:
Does the scala fmt require you to write this comment this way? I mean it would be better if it were written the system verilog way. Vanilla SV users like me prefer to see signals laid out like this so we can easily read.

```suggestion
  * module VerilogMemset #( 
  *        userCsrNum = 1,
  *        dataWidth = 512
  * ) (
  * input logic clk,
  * input logic rst_n,
  * output logic ext_data_i_ready,
  * input logic ext_data_i_valid, 
  * input logic [dataWidth-1:0] ext_data_i_bits,
  * input logic ext_data_o_ready,
  * output logic ext_data_o_valid,
  * output logic [dataWidth-1:0] ext_data_o_bits,
  * input logic [31:0]ext_csr_i_0,
  * input logic ext_start_i,
  * output logic ext_busy_o 
  * );
  *  Several reminders for developers:
```